community/2017-events/12-dev-summit/ContribSummitInformation.md

```diff
@@ -0,0 +1,43 @@
+# 2017 Kubernetes Contributor Summit Information 
+
+## What:
+The Contributor Summit provides an avenue for Kubernetes contributors to connect face to face and mindshare about future community development and community governance endeavors.
+
+In some sense, the summit is a real-life extension of the community meetings and SIG meetings.
+
+## When:
+December 5th, 2017
+Before KubeCon
+Time TBA
+
+## Where:
+Austin Convention Center
+500 E Cesar Chavez St, 
+Austin, TX 78701
+(same location as KubeCon)
+
+## Invitations:
+Seleted attendees are Kubernetes upstream contributors including SIG members, top contributors, and attendee lottery winners.
+We realize that this is not the best approach now that our community is growing in exponential ways. In 2018, we will be unfolding a new strategy for bringing our contributors together so all voices can be heard. 
+
+Invitees will be notified as soon as possible. [section to be updated with further detail]
+
+## Format:
+"loosely-structured unconference"
+Rather than speakers and presentations, we will have moderators/facilitators and discussion topics, alongside all-day completely unstructured hacking.
+
+The discussion sessions will be in an open fishbowl format — rings of chairs, with inner rings driving the discussion — where anyone can contribute. The various sessions will be proposed and voted on by the community in the weeks leading up to the event. This allows the community to motivate the events of the day without dedicating precious day-of time to choosing the sessions and making a schedule.
+
+There will be 3 rooms dedicated to these sessions running in parallel all day. Each session should last between 45 minutes and an hour.
+
+Then, there will be 2 smaller rooms for hacking / unstructured discussion all day. 
+
+## Call for Topics
+Invites will include an area to list your proposed topic to lead for a potential session. Schedule will be announced at least one week before the event. 
+
+## Desired Outcomes
```

Comment:
Did you intend to set this section to be a list? If so, you should add `- ` (or `* `) at the beginning of each sentence. If not, you should add period at the end of each sentence. :)

Reply:
fixed - thank you for pointing it out :)